docs/sources/flow/getting-started/migrating-from-prometheus.md

```diff
@@ -0,0 +1,183 @@
+
+* Have an existing Prometheus configuration.
+* Have a set of Prometheus applications ready to push telemetry data to
+  Grafana Agent.
+* Be familiar with the concept of [Components][] in Grafana Agent flow mode.
+
+[Components]: {{< relref "../concepts/components.md" >}}
+[convert]: {{< relref "../reference/cli/convert.md" >}}
+[run]: {{< relref "../reference/cli/run.md" >}}
+[Start the agent]: {{< relref "../setup/start-agent.md" >}}
+[Flow Debugging]: {{< relref "../monitoring/debugging.md" >}}
+[debugging]: #debugging
+
+## Convert a Prometheus configuration
+
+In order to fully migrate your configuration from [Prometheus] to Grafana Agent
+Flow, your Prometheus configuration must be converted into a Grafana Agent flow
+mode configuration. This will enable you to modify it going forward as a flow
+configuration and take full advantage of the many additional features available
+in Grafana Agent flow mode.
+
+> In this task, we will use the [convert][] CLI command to output flow
+> configuration from a Prometheus configuration.
+
+1. Execute the following:
+
+    ```bash
+    grafana-agent convert --format=prometheus --output=OUTPUT_CONFIG_PATH INPUT_CONFIG_PATH
+    ```
+  
+    1. Replace `INPUT_CONFIG_PATH` with the full path to the Prometheus configuration.
+    2. Replace `OUTPUT_CONFIG_PATH` with the full path to output the flow configuration.
+
+2. [Start the agent][] in flow mode using the new flow configuration from `OUTPUT_CONFIG_PATH`:
+
+The following example demonstrates converting a Prometheus configuration:
+
+```yaml
+global:
+  scrape_timeout:    45s
+
+scrape_configs:
+  - job_name: "prometheus"
+    static_configs:
+      - targets: ["localhost:12345"]
+
+remote_write:
+  - name: "grafana-cloud"
+    url: "https://prometheus-us-central1.grafana.net/api/prom/push"
+    basic_auth:
+      username: USERNAME
+      password: PASSWORD
+```
+
+Execute:
+
+```bash
+grafana-agent convert --format=prometheus --output=OUTPUT_CONFIG_PATH INPUT_CONFIG_PATH
+```
+
+The contents of the new flow configuration:
+
+```river
+prometheus.scrape "prometheus" {
+	targets = [{
+		__address__ = "localhost:12345",
+	}]
+	forward_to     = [prometheus.remote_write.default.receiver]
+	job_name       = "prometheus"
+	scrape_timeout = "45s"
+}
+
+prometheus.remote_write "default" {
+	endpoint {
+		name = "grafana-cloud"
+		url  = "https://prometheus-us-central1.grafana.net/api/prom/push"
+
+		basic_auth {
+			username = "USERNAME"
+			password = "PASSWORD"
+		}
+
+		queue_config {
+			capacity             = 2500
+			max_shards           = 200
+			max_samples_per_send = 500
+		}
+
+		metadata_config {
+			max_samples_per_send = 500
+		}
+	}
+}
+```
+
+### Debugging
+
+1. If Prometheus configuration is provided that cannot be converted,
+   diagnostic information is printed to `stderr`. You can bypass
+   any non-critical issues and output the flow configuration using best
+   effort conversion by including the `--bypass-errors` flag.
```

Comment:
```suggestion
1. If the convert command cannot convert a Prometheus configuration,
   diagnostic information is printed to `stderr`. You can bypass
   any non-critical issues and output the flow configuration using a 
   best- effort conversion by including the `--bypass-errors` flag.
```

Reply:
I updated `best- effort` to `best-effort` in a follow up commit